### ISSUE TITLE
Disable version check

### ISSUE THREAD

**Issue by romestylez** [v10networkscom/saltychat-docs#45]
Hi,
is there a way to disable the version check at Saltychat ?

Im using version 2.3.6 as the new version 3 isnt working on my server but i cant disable the version check popup when starting Teamspeak.

**Comment by xPiwel** [comment]
Update the server & your client would be an easier way of solving this ?

**Comment by romestylez** [comment]
Its not my server. When im using the newest version of SaltyChat i can hear people close to me but they cant hear me. Teamspeak and in-Game it reconizes that im talking but yeah others cant hear me.

Thats why im using an old version and would like to have an option to disable the version check.

**Comment by PietsHost** [comment]
> 
> Thats why im using an old version and would like to have an option to disable the version check.

Exactly the same problem for me and a lot of other ppl on our TS.. still wondering why that happens, but the only solution right now as a downgrade to 2.3.6.

**Comment by romestylez** [comment]
Yep a lot of people having this problem ! You can hear everyone but they cant hear you even the TS3 indicator and the in-game hud show that youre talking.

**Comment by cordamarcel** [comment]
someone have solved this?

**Comment by WulfmanGer** [comment]
I play RedM and we use SaltyChat 2.3.6 here. Newer versions are incompatible. The constant request whether I want to update is annoying ... is there any way to deactivate it? ;) DNS-Resolve blocking? gaming.v10networks.com <- is this the update server for Salty-Chat (i.e. only for this)?

**Comment by Gunner76th** [comment]
For anyone wanting to disable the notification about Salty Chat being out of date: You need to edit your 
%WINDIR%/System32/drivers/etc/hosts and add the following line to the bottom: 127.0.0.1	gaming.v10networks.com
Additionally, if you are on Linux you need to make your edit in /etc/hosts

I have just tested this and confirmed that the plugin is no longer able to check if an update is available and thus does not pop up the update notification. 

Here is the link to a reddit about editing the hosts file to turn off any updates for Teamspeak itself: https://www.reddit.com/r/teamspeak3/
